src/templates/legislation-changes/2016-the-year-of-volatility/index.hbs

```diff
@@ -0,0 +1,68 @@
+
+      <div>
+        <div>
+          <a href="../"><img src="/assets/img/arrow-left.svg" alt="" /></a>
+          <!-- <a href="#"><img src="/assets/img/arrow-left-hover.svg" alt="" /></a> -->
+          <h4>Back</h4>
+        </div>
+
+        <h2>2016 The Year of Volatility</h2>
+        <p>29 February 2016</p>
+        <img src="/assets/img/legislation-changes/keep-informed-article-market-sml.jpg" width="560" height="356" alt="" />
+
+        <p> Given the relentless media coverage of the issue, it will come as no surprise that markets have been in a state of volatility since the start of the year. This volatility comes off the back of the second half of 2015, seeing a 10% reduction in
+          the value of the Australian market. So all in all, not a market that is causing investors to sleep without a tad of anxiety. </p>
+        <p> Having mentioned the media, let’s not forget that good news doesn’t sell papers, make ratings or lead to web site hits. So, the abundance of bad news has been a boon for the news cycle. </p>
+        <p> So, where to from here? There is no doubt that the world is still grappling with the ongoing issues that caused the financial crisis of 2008. The GFC, as it is commonly known, was fuelled by debt which, in the main, remains. In the regulators’
+          attempts to avoid financial Armageddon, more debt (cheap debt) was introduced. Recently there has been commentary about $200 trillion of debt, which will never be repaid. Whilst this is a wonderful headline, it fails to mention that with current
+          low interest rates, debt repayment doesn’t make sense. We must, as a result, focus on borrower’s capacity to meet their interest obligations and, in the majority of cases, they are coping. </p>
+        <p> Central banks have printed money to kick start economic activity and lift inflation. Over time, this will dilute the real value of all this debt. But falling commodity prices around the world have acted as a foil for this desired inflation. It
+          has also sharpened the focus on the ability of commodity revenue dependant emerging economies, to service their debt. This is further complicated by the rising US dollar, given much of their debt is US denominated. </p>
+        <p> These are all reasonable arguments to justify some anxiety in markets. But let’s reset. The US economy has turned a corner. Housing has recovered, the unemployment rate is less than 5% (which purists would argue is full employment), the Federal
+          Reserve has had the confidence to raise interest rates (albeit only once at this stage) and company profits are growing. Europe appears finally, to be getting its act together and may well yet “muddle through”. </p>
+        <p> China is in transition and its economic growth is slowing, however, there are still hundreds of millions to the urbanised. It is fair to say China’s internal consumer demand is yet to gain momentum, and their government continues a “whatever it
```

Comment:
It's not a biggy, but why do the `<p>` start and end with spaces? Perhaps beautify the page? We shouldn't have them there.